packages/headless/src/controllers/query-error/headless-query-error.ts

```diff
@@ -2,21 +2,32 @@ import {Engine} from '../../app/headless-engine';
 import {SearchSection} from '../../state/state-sections';
 import {buildController} from '../controller/headless-controller';
 
+export type QueryError = ReturnType<typeof buildQueryError>;
 /**
- * `QueryError` controller allows to retrieve information about the current error returned by the search API, if any.
+ * A scoped and simplified part of the headless state that is relevant to the `QueryError` controller.
  */
-export type QueryError = ReturnType<typeof buildQueryError>;
 export type QueryErrorState = QueryError['state'];
 
+/**
+ * `QueryError` controller allows to retrieve information about the current error returned by the search API, if any.
+ */
```

Comment:
In the other controllers, we've put this annotation on the actual type (i.e., above line 5)